### DISCUSSION TITLE
Customizable Masked TextField - Doesn't work react-hook-form yup validation

### DISCUSSION THREAD

**Discussion by mitrichy20000** [microsoft/fluentui#26473] [General] [answered]
My inputs
`<Controller
                name="min"
                control={control}
                render={({ field: { onChange, value,ref } }) => (
                <MaskedTextField
                 maskFormat={{
                    "*": /[0-9.,-]/,
                 }}
      value={value}
      onChange={onChange}
      inputRef={ref}
      className="inputStyle"
      />
      )}   
        />`

hook-form
`` const {
    control,
    handleSubmit,
    watch,
    setValue,
    reset,
    register,
    formState: { errors },
  } = useForm({
    mode: "onBlur",
    resolver: yupResolver(validationSchema),
  });``

Mask field onBlur check hook form validation.

**Comment by layershifter** [comment, marked as answer]
@mitrichy20000 you have already created an issue, #26474. Please provide more details there, so we could check.
